<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QTabWidget" name="settings_tabs">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="units_tab">
      <attribute name="title">
       <string>Units</string>
      </attribute>
      <layout class="QVBoxLayout" name="units_tab_layout">
       <item>
        <widget class="QLabel" name="restart_note_label">
         <property name="text">
          <string>A restart may be necessary for changes to take effect.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="unit_system_layout">
         <item>
          <widget class="QLabel" name="system_label">
           <property name="text">
            <string>System</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="unit_system">
           <item>
            <property name="text">
             <string>Metric</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Imperial</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="custom_units_heading">
         <property name="text">
          <string>Custom Units</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="servers_tab">
      <attribute name="title">
       <string>Servers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ServersTable" name="server_table">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_server_button">
         <property name="text">
          <string>Add Server</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="settings_buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServersTable</class>
   <extends>QTableWidget</extends>
   <header>activate/app/tables.h</header>
   <slots>
    <slot>add_row()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>settings_buttons</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_buttons</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_server_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>settings</receiver>
   <slot>add_server()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_server()</slot>
 </slots>
</ui>
